README.md

```diff
@@ -132,6 +132,98 @@ In order to support atomify turtles all the way down, you can also specify your
 
 For detailed information on configuring Rework plugins in package.json see the [relevant section](https://github.com/Techwraith/atomify-css#packagejson-config) of the atomify-css README.
 
+## Mini Tutorial
+### How to build a re-usable component
+(__Warning:__ _The following example is only interesting from a technical and not at all from an esthetical perspective)_
```

Comment:
That's just stupid me being non-native speaker, trying to guess what a certain word would be in english xD
In german, "ästhetisch" means something along the lines of "gifted with a sense of beauty".
But the mini tutorial produces an ugly non-sensical component, which just demos the inner workings of atomify a bit, ...so there is room for improvement here :-)